content/6_Advanced/Parallel_Binary_Search.mdx

```diff
@@ -0,0 +1,132 @@
+---
+id: parallel-binary-search
+title: 'Parallel Binary Search'
+author: Muaath Alqarni
+description: 'Answering multiple binary search queries parallelly'
+prerequisites:
+  - binary-search
```

Comment:
maybe D&Q too?

Reply:
you can do PBS iteratively, so it's not a request I think.

Reply:
the idea is similar to D&Q no? Iterative implemention doesnt mean the idea isnt fundamentally s

Reply:
it's pretty similar yeah